VSS.md

```diff
@@ -0,0 +1,518 @@
+## RFC: 8
+
+## Status: Proposed
```

Comment:
This should be markdown metadata (frontmatter) and the RFC number should be the PR number.

```suggestion
---
RFC: 15
Status: Proposed
---
```